tools/cli/src/utils.rs

```diff
@@ -32,13 +33,36 @@ fn run_command<T: Into<Stdio>>(
     mut command: Command,
     stdout_config: T,
 ) -> Result<String, anyhow::Error> {
-    let process = command.stdout(stdout_config).spawn()?;
+    let process = command.stdout(stdout_config).spawn().map_err(|e| {
+        anyhow::anyhow!(
+            r#"cannot run "{}": {}"#,
+            command.get_program().to_string_lossy(),
+            e
+        )
+    })?;
 
     let output = process.wait_with_output()?;
     if !output.status.success() {
-        anyhow::bail!("failed to execute, exited with {}", output.status)
+        anyhow::bail!(
+            r#"command "{}" exited with {}"#,
+            PrintCommand(&command),
+            output.status
+        )
     }
 
     let stdout = String::from_utf8_lossy(&output.stdout).into_owned();
     Ok(stdout)
 }
+
+struct PrintCommand<'c>(&'c Command);
+
+impl<'c> std::fmt::Display for PrintCommand<'c> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0.get_program().to_string_lossy())?;
```

Comment:
It would be nice to have some escaping.  One can collect the strings into an vec and output as Debug.  It will be not as elegant to read, but at least clear.